.github/workflows/pr.yaml

```diff
@@ -36,18 +36,39 @@ jobs:
           DOCKER_HUB_ACCESS_TOKEN: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
           DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
         run: bin/build-browser-tests
-      - name: Start localstack
-        run: docker-compose -f browser-test/browser-test-compose.yml up -d localstack
-      - name: Wait for localstack ready
-        run: bin/localstack/wait http://localhost:6645
-      - name: Build and run the rest of test stack
-        run: sh -c "docker-compose -f browser-test/browser-test-compose.yml up > .dockerlogs &"
-      - name: Run browser tests
+      - name: Bring up test env with Localstack
+        run: bin/run-browser-test-env --aws --ci
+      - name: Run browser tests with Localstack
         run: bin/run-browser-tests-ci
       - name: Print logs on failure
         if: failure()
         run: cat .dockerlogs
 
+  run_browser_tests_azure:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check out pr branch
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build test app container
+        env:
+          DOCKER_BUILDKIT: 1
+        run: docker build -t civiform --cache-from docker.io/civiform/civiform:latest ./
+      - name: Build browser testing container
+        env:
+          DOCKER_BUILDKIT: 1
+          DOCKER_HUB_ACCESS_TOKEN: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+          DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
+        run: bin/build-browser-tests
+      - name: Bring up test env with Azurite
+        run: bin/run-browser-test-env --azure --ci
+      - name: Run browser tests with Azurite
```

Comment:
Suggestion (non-blocking): I'm wondering if, for now, we should say something like "run dev file upload browser test with Azurite" just for clarity, until we have all browser tests working, then we can clean this up to. Would that make sense?

Reply:
good call!